src/bundle/Resources/public/js/scripts/core/dropdown.js

```diff
@@ -13,11 +13,11 @@
         }
 
         show() {
+            super.show();
+
             if (this.dropdown.container.classList.contains('ibexa-dropdown--disabled')) {
-                return;
+                super.hide();
```

Comment:
So now it will show and then hide. This may cause some flickering issues.

Reply:
Currently, there is no other way to do this (apart from overriding _isHovered). I created an issue in the bootstrap package https://github.com/twbs/bootstrap/issues/39861